server/index.js

```diff
@@ -1,24 +1,47 @@
-const express = require("express");
-const mongoose = require("mongoose");
-
-const compile = require("./routes/api/compile");
-const zip = require("./routes/api/zip");
-const generate = require("./routes/api/generate");
+const express = require('express');
+const mongoose = require('mongoose');
 
+const compile = require('./routes/api/compile');
+const zip = require('./routes/api/zip');
+const generate = require('./routes/api/generate');
+const upload = require('express-fileupload');
 const app = express();
 
 //Database config goes here
 
 //Database connection goes here
-
-app.get("/", (req, res) => {
-  res.send("Hello World");
+app.use(upload());
+app.get('/', (req, res) => {
+	res.send('Hello World');
+});
+app.get('/fileupload', function(req, res) {
+	console.log(__dirname);
+	res.sendFile(__dirname + '/index.html');
+});
+app.post('/upload', function(req, res) {
+	console.log(req.files);
+	var file = req.files.upfile;
+	if (file) {
+		var filename = file.name;
+		var uploadpath = __dirname + '/uploads/' + filename;
```

Comment:
I think we have to check if `/uploads` exists or not too.
To avoid this:
![image](https://user-images.githubusercontent.com/21199234/55647897-7e121480-57fc-11e9-9b32-b84ba1963310.png)